server.go

```diff
@@ -296,6 +368,7 @@ func (s *Server) HandleTracePacket(packet []byte) {
 	// Unlike metrics, protobuf shouldn't have an issue with 0-length packets
 	if len(packet) == 0 {
 		log.Error("received zero-length trace packet")
+		return
```

Comment:
I'm not sure either… maybe @aditya-stripe can clarify?

I skipped it because in testing I was getting `SSFSample`s with no `Trace` and a resulting panic. Since the thing has no content, there't no real point in processing. Hence the empty, but maybe there's more to this? (Also @aditya-stripe mentioned when were on on a pairing session that doing so might be a good idea…)